developer/src/inst/kmdev.wxs

```diff
@@ -257,35 +252,29 @@
       </Component>
 
       <Component Directory="FontsFolder">
-        <File Name='keymanweb-osk.ttf' TrueType='yes' Source='..\..\developer\tike\xml\layoutbuilder\keymanweb-osk.ttf' KeyPath="yes" />
-      </Component>
-
-      <!-- Redistributable files -->
-
-      <Component Directory="dirRedistSetup">
-        <File Name="setup.exe" Source="..\..\..\bin\desktop\setup-redist.exe" KeyPath="yes" />
```

Comment:
I have removed the unsigned Keyman for Windows setup.exe that used to be distributed as part of Keyman Developer from the Developer build and it is now in the Windows release build as setup-redist.exe, for use in bundling (see the Keyman for Windows TC build configuration)